gui_qt/cli_runner.py

```
@@ -0,0 +1,136 @@
+        """Start the CLI command.
+
+        Example:
+            runner.run("/path/to/gemini_translate_batch.py", ["doctor"])
+            runner.run(..., ["build", "--display-name", "foo"])
+        """
+        if self._proc is not None and self._proc.state() == QProcess.ProcessState.Running:
+            self.kill()
+
+        script = Path(script_path).resolve()
+        if not script.exists():
+            self.error.emit(f"找不到命令行脚本：{script}")
```

Ruff (0.15.17)

[warning] String contains ambiguous `：` (FULLWIDTH COLON). Did you mean `:` (COLON)?

(RUF001)